CHANGELOG.rst

```diff
@@ -203,6 +203,26 @@ Improved Documentation
 - `#5416 <https://github.com/pytest-dev/pytest/issues/5416>`_: Fix PytestUnknownMarkWarning in run/skip example.
 
 
+pytest 4.6.4 (2019-06-28)
```

Comment:
Did you move the notes to this position manually? Just curious.

Reply:
yeah the conflict was nasty so I just copied it out of the diff via `git show 4.6.4 -- CHANGELOG.rst | grep '^+'`

Reply:
I figured. 😅